src/probnum/randvars/_random_variable.py

```diff
@@ -122,9 +117,10 @@ def __init__(
         self,
         shape: ShapeArgType,
         dtype: DTypeArgType,
-        random_state: RandomStateArgType = None,
         parameters: Optional[Dict[str, Any]] = None,
-        sample: Optional[Callable[[ShapeType], _ValueType]] = None,
+        sample: Optional[
+            Callable[[ShapeType, Optional[RandomStateArgType]], _ValueType]
```

Comment:
`RandomStateType` instead of `Optional[RandomStateArgType]` for the second argument